app/upgrades/v19/upgrades.go

```diff
@@ -54,24 +55,24 @@ func CreateUpgradeHandler(
 			logger.Error("error removing outposts", "error", err)
 		}
 
-		// bondDenom := sk.BondDenom(ctx)
-
-		// var wevmosContract common.Address
-		// switch {
-		// case utils.IsMainnet(ctx.ChainID()):
-		// 	wevmosContract = common.HexToAddress(erc20types.WEVMOSContractMainnet)
-		// case utils.IsTestnet(ctx.ChainID()):
-		// 	wevmosContract = common.HexToAddress(erc20types.WEVMOSContractTestnet)
-		// default:
-		// 	panic("unknown chain id")
-		// }
-
-		// ctxCache, writeFn = ctx.CacheContext()
-		// if err = RunSTRv2Migration(ctxCache, logger, ak, bk, erc20k, ek, wevmosContract, bondDenom); err == nil {
-		// 	writeFn()
-		// } else {
-		// 	logger.Error("error running strv2 migration", "error", err)
-		// }
+		bondDenom := sk.BondDenom(ctx)
+
+		var wevmosContract common.Address
+		switch {
+		case utils.IsMainnet(ctx.ChainID()):
+			wevmosContract = common.HexToAddress(erc20types.WEVMOSContractMainnet)
+		case utils.IsTestnet(ctx.ChainID()):
+			wevmosContract = common.HexToAddress(erc20types.WEVMOSContractTestnet)
+		default:
+			panic("unknown chain id")
```

Comment:
Would that throw an error for staging?

Reply:
I don't believe so - this shouldn't be executed in SagaOS